src/replay_stage.rs

```diff
@@ -328,24 +349,28 @@ impl ReplayStage {
                                 )
                             };
 
-                            if my_id == leader_id || my_id == last_leader_id {
-                                to_leader_sender
-                                    .send(TvuRotationInfo {
-                                        bank: Bank::new_from_parent(&bank, &leader_id),
-                                        last_entry_id: *last_entry_id.read().unwrap(),
-                                        slot: next_slot,
-                                        leader_id,
-                                    })
-                                    .unwrap();
-                            } else if leader_id != last_leader_id {
-                                // TODO: Remove this soon once we boot the leader from ClusterInfo
-                                cluster_info.write().unwrap().set_leader(leader_id);
+                            {
+                                let mut bank_forks = bank_forks.write().unwrap();
+                                bank_forks
```

Comment:
I think we want the bank_id to equal the slot_id for that bank? Currently I think this will cause the bank_id to be one behind the slot_id.